src/codegen/flags.rs

```diff
@@ -256,7 +256,9 @@ impl FromGlib<{sys_crate_name}::{ffi_name}> for {name} {{
     unsafe fn from_value(value: &'a {gvalue}) -> Self {{
         {assert}from_glib({glib}(value.to_glib_none().0))
     }}
-}}",
+}}
```

Comment:
Same for the line below this one (gh doesn't let me create a new conversation on this line :'( ).